Translation Corrections (#75)
07th-mod/higurashi-console-arcs: Update/some_tips_081.txt

	if (GetGlobalFlag(GADVMode)) {
		OutputLineAll("", NULL, Line_ContinueAfterTyping);
		OutputLine(NULL, "そもそも尾形に限らず、あの学校に好きな先生なんていない。私は、幼馴染の珠子や暁が一緒だから入学しただけで、進学校ならではの偏差値至上主義ってやつには正直、ウンザリしていた。",
				NULL, "<size=-2>First off, it's not just Ogata. There are no really good teachers at my school. For me, who enrolled in this school just to be with Tamako and Akira, the expectations associated with being in an advanced school have been sickening.", Line_Normal);
	} else {
		OutputLine(NULL, "そもそも尾形に限らず、あの学校に好きな先生なんていない。私は、幼馴染の珠子や暁が一緒だから入学しただけで、進学校ならではの偏差値至上主義ってやつには正直、ウンザリしていた。",

> 				NULL, "First off, it's not just Ogata, there are no really good teachers at my school. Ever since Tamako, Akira and I enrolled in this school together, the expectations associated with being in an advanced school have been sickening.", Line_Normal);

Reviewer: ```suggestion
				NULL, "First off, it's not just Ogata. There are no really good teachers at my school. For me, who enrolled in this school just to be with Tamako and Akira, the expectations associated with being in an advanced school have been sickening.", Line_Normal);
```

			NULL, "\"Ahahaha, ...is that right?", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s20/37/vtt_chisa3013", 256, TRUE);
	OutputLine(NULL, "　まぁ来日するのも今回限りじゃないみたいだし、次の機会を待てばいいって」",
			NULL, " Well, this isn't the only time they'll be coming to Japan, so I'll just wait for my next chance.\"", Line_Normal);
	ClearMessage();

//千紗登rvS20/37/VTT_chisa3014.「それに裕美、ものすごく行きたがっていたのに抽選外れて落ち込んでたから、彼女に恩を売って一石二鳥ってとこかな～？kvS20/37/VTT_chisa3015.　くっくっくっ…！」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 37, "ps3/s20/37/vtt_chisa3014", 256, TRUE);
	OutputLine(NULL, "「それに裕美、ものすごく行きたがっていたのに抽選外れて落ち込んでたから、彼女に恩を売って一石二鳥ってとこかな～？",

> 			NULL, "\"Plus, Yumi was really depressed about not scoring a ticket, so now she'll owe me when I give her the ticket. Killing two birds with one stone, huh?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Plus, Yumi was really depressed about not scoring a ticket, so now she'll owe me when I give it to her. Killing two birds with one stone, huh?", Line_WaitForInput);
```

			NULL, "\"...", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s20/00/vtt_saekif3019", 256, TRUE);
	OutputLine(NULL, "お前の気持ちは、よくわかる。…わかっているつもりだ。",
			NULL, " I know how you feel. ...At least, I'd like to think I do.", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s20/00/vtt_saekif3020", 256, TRUE);
	OutputLine(NULL, "だがな、千紗登——」",

> 			NULL, " But Chisato...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " But, Chisato...\"", Line_Normal);
```

//r自分の触れられたくない場所をなでられたような不快感に、私は鋭く言葉を差し向ける。…やっぱり親子だけあって、それだけで父さんはその意図を理解してくれた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "自分の触れられたくない場所をなでられたような不快感に、私は鋭く言葉を差し向ける。…やっぱり親子だけあって、それだけで父さんはその意図を理解してくれた。",

> 			NULL, "I lashed out at him prodding at things I didn't want touched upon. ...Being my father, that was enough for him to realize he'd rubbed me the wrong way.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I lashed out at him for prodding at things I didn't want touched. ...Being my father, that was enough for him to realize he'd rubbed me the wrong way.", Line_Normal);
```
